lib/srv/desktop/rdp/rdpclient/src/rdpdr/mod.rs

```diff
@@ -408,6 +408,11 @@ impl Client {
                     // https://github.com/FreeRDP/FreeRDP/blob/511444a65e7aa2f537c5e531fa68157a50c1bd4d/winpr/libwinpr/file/file.c#L781
                     if rdp_req.create_disposition == flags::CreateDisposition::FILE_SUPERSEDE {
                         // If the file already exists, replace it with the given file. If it does not, create the given file.
+                        if res.err_code == TdpErrCode::Nil {
```

Comment:
I'm confused, don't you have an error code for already exists? (`TdpErrCode::AE`)

Why are you not checking for that?